paddle/fluid/operators/fill_constant_op.cc

```diff
@@ -63,7 +63,7 @@ class FillConstantOp : public framework::OperatorWithKernel {
   }
 
   framework::OpKernelType GetExpectedKernelType(
-      const framework::ExecutionContext& ctx) const override {
+      const framework::ExecutionContext &ctx) const override {
```

Comment:
Maybe leave all above `&` as it is. Cause you did not actually change anthing right ?

Reply:
Hi, its just clang format. I have not done it manually, but I can revert it if you want

Reply:
Hi, you mean it is pre-commit automatically changed ? If it is pre-commit changed, then you may don't have to change anything. Thanks !